term.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/pkg/term/termios"
 )
 
+const (
+	NONE     = iota // flow control off
```

Comment:
huh - i see them show up in godoc locally as is. do they show up anywhere other than the `Constants` section on the package page?

happy to move them either way